amitbisht-cg/demo-cg
# Added New Product History Page

## Description of the Pull Request
New feature code for Product catalog in the web UI

## Changes Made
- added new template file to Pull Requests
- Created new API for product history
- added DB connections details for zeta non-live

## Dependencies
- Before deploying the new product page, Database server needs to be created and connected to API layer.

## Associated Work Item
[34533](https://github.com/amitbisht-cg/demo-cg/compare/testbranch...main)

## Type of change

## Testing and Validations
- [x] Unit Testing Coverage?

### Steps for Testing

1. Login into Cart Portal
2. Go to Product History page from navigation Menu
3. Click on Search History
4. Select the date "past 6 months" and Click on "Search" Button
5. Last 6 months purchase history is displayed

## Screenshots or Demonstration
<img width="376" alt="image" src="https://user-images.githubusercontent.com/113919526/218502360-850e6a2c-1560-48ce-933a-8a1847d012ed.png">

## Checklist
- [x] I have written/updated documentation or README for the changes.
- [ ] I have written unit tests for the changes, if applicable.
- [x] I have tested the changes in my local environment, and they work as expected.
- [ ] I have verified that the changes do not introduce any new bugs or regressions.
- [x] I have reviewed the changes and I am confident that they are ready to be merged.

## Additional context
None
